src/components/CodeHighlight/LogColorizer.js

```diff
@@ -0,0 +1,194 @@
+          <div className={css.filter}>
+            <span className={css.filter__label}>
+              <FormattedMessage id="ui-data-import.logColorizer.filter.label" />:
+            </span>
+            <ButtonGroup
+              data-test-logs-filter
+              fullWidth
+              role="tablist"
+            >
+              {filterOptions.map(option => (
+                <Button
+                  role="tab"
+                  aria-selected={currentFilter === option.id}
+                  id={`option-${option.id}`}
+                  key={`option-${option.id}`}
+                  buttonStyle={currentFilter === option.id ? 'primary' : 'default'}
+                  className={css.filter__button}
+                  data-test-logs-filter-option={option.id}
+                  marginBottom0
+                  onClick={() => setCurrentFilter(option.id)}
+                >
+                  <FormattedMessage id={option.caption} />
+                </Button>
+              ))}
+            </ButtonGroup>
+          </div>
+          {showThemes && (
+            <div className={css.themes}>
+              <span className={css.themes__label}>
+                <FormattedMessage id="ui-data-import.logColorizer.themes.label" />:
+              </span>
+              <Select
+                onChange={e => setCurrentTheme(e.target.value)}
+                value={currentTheme}
+                marginBottom0
+              >
+                {themesPresent.map(item => (
+                  <option
+                    key={item.id}
+                    value={item.id}
+                  >
+                    {item.caption}
+                  </option>
+                ))}
+              </Select>
+            </div>
+          )}
+        </div>
+      )}
+      <pre
+        id="logs-pane"
+        className={currentTheme}
+      >
+        {codePortion.map((item, i) => {
+          const codeString = JSON.stringify(item, null, 2);
+          const hasError = codeString.includes('error');
+          const showErrors = hasError && OPTIONS.ERRORS === currentFilter;
+          const showInfo = !hasError && OPTIONS.INFO === currentFilter;
+          const showAll = OPTIONS.ALL === currentFilter;
+
+          return (showAll || showErrors || showInfo) && (
+            <CodeHighlight
+              key={`snippet-${i}`}
+              code={codeString}
+              language={language}
+              theme={currentTheme}
+              className={hasError ? hlTheme.error : hlTheme.info}
+            />
+          );
+        })}
+      </pre>
+    </Fragment>
+  );
+});
+
+LogColorizer.propTypes = {
+  code: PropTypes.any,
+  language: PropTypes.string,
+  theme: PropTypes.string,
+  toolbar: PropTypes.shape({
+    visible: PropTypes.bool,
+    message: PropTypes.any,
+    showThemes: PropTypes.bool,
+  }),
+};
+
+LogColorizer.defaultProps = {
+  code: '',
+  language: LANGUAGES.RAW,
+  theme: THEMES.STALKER,
```

Comment:
I think the light theme will be a better choice as the application is mostly light

Reply:
Done